apps/angular/src/app/features/anime/table/anime-table.component.ts

```diff
@@ -30,12 +73,27 @@ export class AnimeTableComponent implements OnInit {
     'status',
   ] as const;
 
+  /** Anime type options map. */
+  public readonly animeTypeOptionsMap = animeTypeOptionsMap;
+
+  @ViewChild(MatSort) private tableSort?: MatSort;
```

Comment:
We usually place a field under decorator like so:
```suggestion
  @ViewChild(MatSort)
  private tableSort?: MatSort;
```